src/test/java/org/broadinstitute/hellbender/utils/UtilsUnitTest.java

```diff
@@ -1,26 +1,36 @@
 package org.broadinstitute.hellbender.utils;
 
 
+import static java.util.Arrays.asList;
+import static org.testng.Assert.assertEquals;
+
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.Lists;
 import com.google.common.collect.Sets;
 import com.google.common.primitives.Ints;
 import htsjdk.samtools.util.Log.LogLevel;
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Locale;
+import java.util.Random;
+import java.util.Set;
+import java.util.TreeSet;
+import java.util.stream.Collectors;
```

Comment:
`import java.util.*;` (looks like you need to set your IDE to collapse imports after 5 from the same package, which is the setting most of us use)

Reply:
Done.